src/components/Log/LogAction.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Divider } from '@mui/material';
+import { LogActionActor } from './LogActionActor';
+import { LogActionItem } from './LogActionItem';
+import { LogAction as LogActionType } from '../../types/log.types';
+
+export const LogAction = ({ actor, organization, items }: LogActionType) => {
+  return (
+    <>
+      <Divider sx={{ bgcolor: 'primary.main' }} />
+      <Box sx={{ display: 'grid', gridTemplateColumns: '1fr auto', gap: '0.5rem' }}>
+        {(!!actor || !!organization) && <LogActionActor actor={actor} organization={organization} />}
+        {items && items.map((item, index) => <LogActionItem {...item} key={index} />)}
```

Comment:
React er ikke så begeistret for at `index` brukes som key, så kanksje kombiner det med en annen verdi?

https://react.dev/learn/rendering-lists

Reply:
Har egentlig ikke noe annet å bruke, da item-objektet kun inneholder en dato, ikon og beskrivelse. Det eneste som kan være unikt (men ikke nødvendigvis) er beskrivelse, men den kan jo bli lang...